torch/autograd/gradcheck.py

```diff
@@ -252,8 +275,8 @@ def check_jacobians_equal(j1, j2, atol):
     return True
 
 
-def combine_jacobian_rows(jacobians_rows, inputs, output):
-    out_jacobians = allocate_jacobians_with_inputs(inputs, output.numel())
+def combine_jacobian_rows(jacobians_rows, inputs, dim):
```

Comment:
nit: dim -> numel or "output_numel" might make more sense? `dim` is an overloaded word, it can mean "the number of dimensions in your tensor, e.g. tensor.dim()" or "the size of dimension X in your tensor, e.g. tensor.shape[X]"